test/nested.js

```diff
@@ -7,20 +7,20 @@ var Nested = protobuf(require('./test.proto')).Nested
 tape('nested encode', function (t) {
   var b1 = Nested.encode({
     num: 1,
-    payload: Buffer.from('lol'),
+    payload: new TextEncoder().encode('lol'),
```

Comment:
we can't yet use TextEncoder like this, old node versions still have it inside `util`

use this instead for now
```js
const TextEncoder = require('ipfs-utils/src/text-encoder.js')
```

Reply:
This is only running in test which are running only on Node v12
https://github.com/ipfs/protons/blob/3b5276f052f2e17c2d806d27cd9a88e156588977/.travis.yml#L8-L9

That have `TextEncoder` / `TextDecoder` globals.

Reply:
@hugomrdias Please let me know if you still want me to import it from `ipfs-utils`. Note that this library does not at the moment depend on `ipfs-utils`.

Reply:
Yes and probably add node 10 cause we support bit upstream

Reply:
Blocked on https://github.com/ipfs/js-ipfs-utils/pull/44